tests/serialization/vector.output

```diff
@@ -1,6 +1,6 @@
 
-DEAL::0 0 0 0 5 5 0 1 2 3 4
+DEAL::0 0 0 0 0 0 5 0 1 2 3 4
 
-DEAL::0 0 0 0 5 5 0 1 2 3 4
+DEAL::0 0 0 0 0 0 5 0 1 2 3 4
```

Comment:
I think it would be possible to keep this compatible: do we try to keep serialized objects valid between different library releases?

Reply:
I would say that we don't need to spend too much effort in keeping them compatible.
The serialized data should only be used within the same program (version).